build/helper/metadata_add_all.py

```diff
@@ -6,9 +6,11 @@
 from .metadata_merge_dicts import merge_dicts
 
 import copy
+import os
 import pprint
 
 pp = pprint.PrettyPrinter(indent=4, width=80)
+pp_persist = pprint.PrettyPrinter(indent=4, width=300)
```

Comment:
only used in one function, declare there in order to reduce its scope.

Reply:
Moved